src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString === '') {
+    return {};
+  }
+
+  const styleArray = sourceString.split(';');
+  const obj = {};
```

Comment:
Consider using a more descriptive variable name instead of 'obj'. For instance, 'stylesObject' or 'cssStyles' could enhance readability and better convey the purpose of the variable.